src/main/java/com/es/phoneshop/model/productSortEnums/SortOrder.java

```diff
@@ -0,0 +1,5 @@
+package com.es.phoneshop.model.productSortEnums;
+
+public enum SortOrder {
+    asc, desc
```

Comment:
Here and in other places: constants in enums should follow constants code conventions and be upper case: https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html

Reply:
done